pkg/config/schema.go

```diff
@@ -119,39 +119,72 @@ func (t *dependsOn2_1) Decode(into mapdecode.Into) error {
 
 type environmentNameValuePair struct {
 	Name  string
-	Value *string
+	Value *environmentValue
+}
+
+// TODO https://github.com/jbrekelmans/kube-compose/issues/40 check whether handling of large numbers is consistent with docker-compose
+// See https://github.com/docker/compose/blob/master/compose/config/config_schema_v2.1.json#L418
+type environmentValue struct {
```

Comment:
Should we also consider taking these as an array of values?

Reply:
I think what you are talking about already works, you mean this right:
```yaml
environment:
- VAR1=v1
- VAR2=v2
```